openhtf/core/test_state.py

```diff
@@ -150,7 +150,7 @@ def __init__(self, test_desc, execution_uid, test_options):
     self.execution_uid = execution_uid
     self.test_options = test_options
 
-  def __del__(self):
+  def cleanup(self):
```

Comment:
So the idea is instead of deleting the test state, you call cleanup which removes the logs record handler... is this effectively dereferencing the last reference to test state, so it will be garbage collected?

I'd say worth a comment or docstring here.

Reply:
Done.